scripts/stork.sh

```diff
@@ -85,11 +83,14 @@ if [[ "$kind" == "module" ]]; then
 fi
 
 if [[ "$kind" == "app" ]]; then
-    (
-        cd $dir
-        git init
-        git add *
-    )
+    # copy clang-format related files
+    mkdir -p $dir/scripts
```

Comment:
I know this isn't the first place in this script where you "write" but what does this script do if you don't have write permissions where you are running (trust me, users do this....). Perhaps that should be added as another up front check?